components/combobox/combobox.jsx

```diff
@@ -254,6 +254,40 @@ const propTypes = {
 	 * Changes styles of the input. Currently `entity` is not supported. _Tested with snapshot testing._
 	 */
 	variant: PropTypes.oneOf(['base', 'inline-listbox', 'readonly']),
+	/**
+	 * Default value of input. Provide uncontroled behaviour
+	 */
+	defaultValue: PropTypes.string,
+	/**
+	 * Provided to input to make it disabled
+	 */
+	disabled: PropTypes.bool,
```

Comment:
Not sure why he added this? We already have `singleInputDisabled`. We would need to figure out what it means to disable the input for the cases where it's not a single select.

Reply:
Please remove. There will never be an all variants disabled prop on a combobox. If folks want to hack with with `input` prop. They can I think.